### PR TITLE
Fstab generator fixes

### PR DESCRIPTION
**Pull request by daandemeyer**

### CODE REVIEW

File: src/fstab-generator/fstab-generator.c [hunk 1]
```diff
@@ -1163,21 +1163,21 @@ static int parse_proc_cmdline_item(const char *key, const char *value, void *dat
...
                 if (proc_cmdline_value_missing(key, value))
                         return 0;
 
-                return free_and_strdup_warn(&arg_usr_what, value);
+                return free_and_strdup_warn(&arg_usr_what, empty_to_null(value));
 
         } else if (streq(key, "mount.usrfstype")) {
 
-                if (proc_cmdline_value_missing(key, value))
+                if (proc_cmdline_value_missing(key, empty_to_null(value)))
```
Comment (yuwata) [inline src/fstab-generator/fstab-generator.c]: Hm? I don't think this is necessary.

Comment (daandemeyer) [reply src/fstab-generator/fstab-generator.c]: Oops

File: src/fstab-generator/fstab-generator.c [hunk 2]
```diff
@@ -1216,6 +1217,14 @@ static int parse_proc_cmdline_item(const char *key, const char *value, void *dat
                         log_warning("Failed to parse systemd.swap switch %s. Ignoring.", value);
                 else
                         arg_swap_enabled = r;
+
+        } else if (streq(key, "systemd.verity")) {
+
+                r = value ? parse_boolean(value) : 1;
+                if (r < 0)
+                        log_warning("Failed to parse verity= kernel command line switch %s. Ignoring.", value);
```
Comment (yuwata) [inline src/fstab-generator/fstab-generator.c]: verity= -> systemd.verity=

**Review by yuwata — APPROVED** [review]

**Comment by yuwata** [comment]
Ah, I forgot one thing. Please update document and mention systemd.verity=.

**Pull request merged**